storage-engine/titan-configuration.md

```diff
@@ -0,0 +1,142 @@
+
++ Method 1: If you have deployed the cluster using TiUP, you can execute the `tiup cluster edit-config ${cluster-name}` command and edit the TiKV configuration file as the following example shows:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+      tikv:
+        rocksdb.titan.enabled: true
+    ```
+
+    Reload the configuration and TiKV will be rolling restarted online:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    tiup cluster reload ${cluster-name} -R tikv
+    ```
+
+    For the detailed command, see [Modify the configuration using TiUP](/maintain-tidb-using-tiup.md#modify-the-configuration).
+
++ Method 2: Directly edit the TiKV configuration file to enable Titan (**NOT** recommended for the production environment).
+
+    {{< copyable "" >}}
+
+    ``` toml
+    [rocksdb.titan]
+    enabled = true
+    ```
+
+After Titan is enabled, the existing data stored in RocksDB is not immediately moved to the Titan engine. As new data is written to the TiKV foreground and RocksDB performs compaction, the values are progressively separated from keys and written to Titan. You can view the **TiKV Details** -> **Titan kv** -> **blob file size** panel to confirm the size of the data stored in Titan.
+
+If you want to speed up the writing process, you can compact data of the whole TiKV cluster manually using tikv-ctl. For details, see [manual compaction](/tikv-control.md#compact-data-of-the-whole-tikv-cluster-manually).
+
+> **Note:**
+>
+> When Titan is disabled, RocksDB cannot read data that has been migrated to Titan. If Titan is incorrectly disabled on a TiKV instance with Titan already enabled (mistakenly set `rocksdb.titan.enabled` to `false`), TiKV will fail to start, and the `You have disabled titan when its data directory is not empty` error appears in the TiKV log. To correctly disabled Titan, see [Disable Titan](#disable-titan-experimental)
+
+## Parameters
+
+To adjust Titan-related parameters using TiUP, refer to [Modify the configuration](/maintain-tidb-using-tiup.md#modify-the-configuration).
+
++ Titan GC thread count.
+
+    From the **TiKV Details** -> **Thread CPU** -> **RocksDB CPU** panel, if you observe that the Titan GC threads are at full capacity for a long time, consider increase the size of the Titan GC thread pool.
+
+    {{< copyable "" >}}
+
+    ```toml
+    [rocksdb.titan]
+    max-background-gc = 1
+    ```
+
++ Threshold of value size.
+
+    When the value written to the foreground is smaller than the threshold, this value is stored in RocksDB; otherwise, this value is stored in the blob file of Titan. Based on the distribution of value sizes, if you increase the threshold, more values are stored in RocksDB and TikV performs better in reading small values. If you decrease the threshold, more values go to Titan, which further reduces RocksDB compactions.
```

Comment:
```suggestion
    When the size of the value written to the foreground is smaller than the threshold, this value is stored in RocksDB; otherwise, this value is stored in the blob file of Titan. Based on the distribution of value sizes, if you increase the threshold, more values are stored in RocksDB and TiKV performs better in reading small values. If you decrease the threshold, more values go to Titan, which further reduces RocksDB compactions.
```